cmd/nfs-provisioner/main.go

```diff
@@ -81,10 +81,6 @@ func main() {
 	// Create the client according to whether we are running in or out-of-cluster
 	outOfCluster := *master != "" || *kubeconfig != ""
 
-	if !outOfCluster && *serverHostname != "" {
-		glog.Fatalf("Invalid flags specified: if server-hostname is set, either master or kube-config must also be set.")
-	}
-
```

Comment:
Any reason for removing this check?

Reply:
Hmmm I don't remember exactly. I think that I added that because I wanted to use the --server-hostname flag to set the host, even when running _in_ the cluster (if I read the code correctly, this flag is only supported when running _outside_ kubernetes?)